Math-Trainer/Math-Trainer/ViewController.swift

```diff
@@ -30,7 +38,24 @@ class ViewController: UIViewController {
         performSegue(withIdentifier: "goToNext", sender: sender)
     }
     
-    @IBAction func unwindAction(unwindSegue: UIStoryboardSegue) { }
+    @IBAction func unwindAction(unwindSegue: UIStoryboardSegue) {
+        if let sourceViewController = unwindSegue.source as? TrainViewController {
+            switch selectedType {
+            case .add:
+                addCount += Int(sourceViewController.count)
+                addCountLabel.text = String(addCount)
+            case .subtract:
+                subtractCount += Int(sourceViewController.count)
+                subtractCountLabel.text = String(subtractCount)
+            case .multiply:
+                multiplyCount += Int(sourceViewController.count)
+                multiplyCountLabel.text = String(multiplyCount)
+            case .devide:
+                devideCount += Int(sourceViewController.count)
+                devideCountLabel.text = String(devideCount)
+            }
+        }
```

Comment:
I think u don't need `devideCount` u can immediately set `devideCountLabel.text = String(sourceViewController.count)`

```
 if let sourceViewController = unwindSegue.source as? TrainViewController {
            let rightAnswersCount = Int(sourceViewController.count)
            switch selectedType {
            case .add:
                addCountLabel.text = rightAnswersCount
            case .subtract:
                subtractCountLabel.text = rightAnswersCount
            case .multiply:
                multiplyCountLabel.text = rightAnswersCount
            case .devide:
                devideCountLabel.text = rightAnswersCount
            }
        }
```

Reply:
addCount, subtractCount, multiplyCount and devideCount store the current number of correct answers

Reply:
got it